package.json

```diff
@@ -202,7 +202,8 @@
     "jest": "workspace:*",
     "jest-environment-node": "workspace:*",
     "psl": "patch:psl@npm:^1.9.0#./.yarn/patches/psl-npm-1.9.0-a546edad1a.patch",
-    "ts-node@^10.5.0": "patch:ts-node@npm:^10.5.0#./.yarn/patches/ts-node-npm-10.9.1-6c268be7f4.patch"
+    "ts-node@^10.5.0": "patch:ts-node@npm:^10.5.0#./.yarn/patches/ts-node-npm-10.9.1-6c268be7f4.patch",
+    "typescript": "~5.5.4"
```

Comment:
this shouldn't be here

Reply:
I agree. It is likely an oversight, there is no dependency on these two packages. I will fix that.